src/default_material.js

```diff
@@ -0,0 +1,43 @@
+import { ATOMIC_COORD_UNITS, units } from "./constants";
+import { LATTICE_TYPE } from "./lattice/types";
+
+export const defaultMaterialConfig = {
+    name: "Silicon FCC",
+    basis: {
+        elements: [
+            {
+                id: 1,
+                value: "Si",
+            },
+            {
+                id: 2,
+                value: "Si",
+            },
+        ],
+        coordinates: [
+            {
+                id: 1,
+                value: [0.0, 0.0, 0.0],
+            },
+            {
+                id: 2,
+                value: [0.25, 0.25, 0.25],
+            },
+        ],
+        units: ATOMIC_COORD_UNITS.crystal,
+    },
+    lattice: {
+        // Primitive cell for Diamond FCC Silicon at ambient conditions
+        type: LATTICE_TYPE.FCC,
```

Comment:
I suspect the fact that we are using LATTICE_TYPE here and in the Lattice, and then import Lattice inside Material could lead to circular dependency which could lead to the problem with tests for https://github.com/Exabyte-io/exabyte-stack/pull/235/files#r805085142

Reply:
okay i will look into this.

Reply:
We saw these same errors prior to me moving default materials out of material.


Previously we imported both LATTICE_TYPE & LATTICE into material.js. Wouldn't that lead to the same circular error?